docs/docusaurus.config.js

```diff
@@ -0,0 +1,140 @@
+// @ts-check
+
+const lightCodeTheme = require('prism-react-renderer/themes/github');
+const darkCodeTheme = require('prism-react-renderer/themes/dracula');
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Mapperly',
+  tagline: 'A .NET source generator for generating object mappings. Without any reflection. Inspired by MapStruct.',
```

Comment:
On a second thought, "Without any reflection" sounds a little weird... Maybe change it to "Does not use reflection"? Or something else